change/@ni-nimble-components-752d1b02-6122-4d2f-b737-809aff6729d2.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
I think it'd be worthwhile to add review and update our Storybook docs on table column sizing. I don't see much to change but it might be helpful to modify the parameters for "Fractional width configuration" and `min-pixel-width` to mention that:
1. fractional width is just about initial size and size when the table is resized. It's not updated when you resize a column
2. min-pixel-width is applied when the table resizes or when a column is resized